code/__HELPERS/_macros.dm

```diff
@@ -11,6 +11,8 @@
 
 #define ismartian(A) istype(A, /mob/living/carbon/martian)
 
+#define ishigherbeing(A) (ismartian(A) || ishuman(A))
```

Comment:
Minor nitpick: since `ishuman` is going to pass more, it'd be a tiny micro optimization to make that the first side of the OR statement.

I realize this is 100% negligible but it still came to my mind and you can change it if you want but I'm not holding it against you either.

Reply:
optimisations are optimisations,